chef/cookbooks/iptables/CONTRIBUTING.md

```diff
@@ -0,0 +1,2 @@
+Please refer to
```

Comment:
remove file, that refers to instructions for other projects.

Reply:
I can remove it, but it came straight from a downloaded zip from https://supermarket.chef.io/cookbooks/iptables -- we should/could probably move to a git submodule approach, but how would that impact your testing workflows?

Reply:
For testing adding submodule checkout is not a problem, it does add some complexity to for users.  If submodules are added, then `README.md` needs to be clear about usage and likely `build.sh` and `build.ps1` will need to be updated to check they have been pulled before attempting to execute.

Reply:
Eh, perhaps for a future PR. I removed CONTRIBUTING.md for now.

Reply:
Would you be okay installing the chefdk  in the testing run in order to use berks for dependency management? Where are the ubuntu test scripts -- or is it just the packer template and provisioning therein?

Reply:
The only current tests for ubuntu are part of the chef recipe here https://github.com/rapid7/metasploitable3/tree/master/chef/cookbooks/metasploitable/test

Adding `chefdk` is likely a bit of overkill for the project at this time.  An additional note, I have found many users that choose to build these VMs themselves have had internet connectivity issues and adding more dependencies on infrastructure not provided by the OS and github is not a favored approach at this time.

Reply:
Hey here's an idea: we could move the vendor cookbooks to their own directory, and include that directory along with the primary metasploitable3 cookbooks. That's how I did it with my project's repo, see [here](https://github.com/deargle/gcp-vuln-server-behing-openvpn). Multiple cookbook paths can be passed to `chef-solo` for provisioning. Users would not have to install `chefdk` or use `berks` -- the vendor cookbooks could still be committed directly to the repo -- but it would be clearly delineated that those cookbooks were vendored. Would you be okay with github-related artifacts like CONTRIBUTING.md to remain in vendor cookbooks if they were in a directory clearly marked as such?